configure.ac

```diff
@@ -4711,7 +4711,7 @@ if test "$PYTHON_CONFIG" != ""; then
   if test $? -ne 0; then
     with_libpython="no"
   fi
-  LIBPYTHON_LDFLAGS="`${PYTHON_CONFIG} --ldflags`"
+  LIBPYTHON_LDFLAGS="`${PYTHON_CONFIG} --ldflags --embed`" || LIBPYTHON_LDFLAGS="`${PYTHON_CONFIG} --ldflags`"
```

Comment:
please redirect stderr of the first command to /dev/null, otherwise an error shows up on all systems where we don't have python 3.8.

Reply:
I have checked `python2-config --unsupported-flag` and `python3-config --unsupported-flag` behaviour on couple of systems. And everywhere `python-config` sent error message to stdout. So on systems with python older than 3.8 error message would be stored in `$LIBPYTHON_LDFLAGS` and not displayed.

On the other hand, future versions of python might correctly give error to stderr if something went wrong unrelated to `--embed` flag.

Therefore it seems to me current variant, without devnulling of stderr, is better.

Do you insist on stderr redirection anyway?

Reply:
That makes sense to me and is also consistent with the other part of python3.8 work